### PR TITLE
Merge Dev to Main for Initial Release

### PR DESCRIPTION
**Pull request by GoldenPavilion**

### CODE REVIEW

File: .byebug_history [hunk 1]
```diff
@@ -1,2 +1,48 @@
 continue
```
Comment (MiniCodeMonkey) [inline .byebug_history:1]: This file should be added to .gitignore

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (MiniCodeMonkey) [inline .vscode/tasks.json:1]: .vscode should also be gitgnored

**Review by MiniCodeMonkey — APPROVED** [review]
Just a couple of files to add to gitignore. Otherwise, all good! 👍

File: .byebug_history (hunk 1 above)
Comment (GoldenPavilion) [reply .byebug_history:1]: Will do!

**Pull request merged**